contracts/src/bin/vendor.rs

```diff
@@ -28,8 +28,11 @@ fn run() -> Result<()> {
         .full()
         .github(
             "BalancerV2Vault",
-            "balancer-labs/balancer-core-v2/v1.0.0-artifacts/\
-             deployments/artifacts/mainnet/Vault.json",
+            "balancer-labs/balancer-v2-monorepo/a3b570a2aa655d4c4941a67e3db6a06fbd72ef09/pkg/deployments/deployed/mainnet/Vault.json",
+        )?
+        .github(
+            "BalancerV2WeightedPool",
+            "balancer-labs/balancer-v2-monorepo/a3b570a2aa655d4c4941a67e3db6a06fbd72ef09/pkg/deployments/extra-abis/WeightedPool.json",
         )?
```

Comment:
This probably requires only vendoring the ABI and should be added to the `vendor.abi_only()` list below :point_down:.